### PR TITLE
Wired up Show Information|Warning|Error Message

### PR DESCRIPTION
**Pull request by dfinke**
This change is [Reviewable](https://reviewable.io/reviews/powershell/powershelleditorservices/244)

### CODE REVIEW

**Comment by daviwil** [comment]
Thanks man!  I'll help you get this finished up when I've got a few spare moments.

File: src/PowerShellEditorServices/Extensions/EditorObject.cs [hunk 1]
```diff
@@ -81,7 +87,7 @@ public void UnregisterCommand(string commandName)
         public EditorContext GetEditorContext()
         {
             return this.editorOperations.GetEditorContext().Result;
-       }
+        }
```
Comment (daviwil) [inline src/PowerShellEditorServices/Extensions/EditorObject.cs]: Extra space sneaked in here ;)

File: src/PowerShellEditorServices/Workspace/ScriptFile.cs [hunk 2]
```diff
@@ -499,26 +499,26 @@ private void ParseFileContents()
...
-                    this.ScriptAst =
-                        Parser.ParseInput(
-                            this.Contents,
-                            this.FilePath,
-                            out this.scriptTokens,
-                            out parseErrors);
-                }
-                else
-                {
+                //if (this.powerShellVersion.Major >= 5 &&
```
Comment (daviwil) [inline src/PowerShellEditorServices/Workspace/ScriptFile.cs]: By the way, any reason for commenting this out?

Comment (dfinke) [inline src/PowerShellEditorServices/Workspace/ScriptFile.cs]: yeah, it didn't compile with it. I forgot to uncomment that.

**Comment by daviwil** [comment]
Thanks a lot for the PR, Doug, I appreciate it!  I apologize for taking so long to get it fixed up and merged.

**Pull request merged**